homeassistant/components/sense/sensor.py

```diff
@@ -161,19 +142,15 @@ class SenseActiveSensor(SensorEntity):
     def __init__(
         self,
         data,
-        name,
-        sensor_type,
         sense_monitor_id,
         variant_id,
         variant_name,
-        unique_id,
-    ):
+    ) -> None:
         """Initialize the Sense sensor."""
-        self._attr_name = f"{name} {variant_name}"
-        self._attr_unique_id = unique_id
+        self._attr_name = f"{ACTIVE_NAME} {variant_name}"
+        self._attr_unique_id = f"{sense_monitor_id}-{ACTIVE_TYPE}-{variant_id}"
```

Comment:
Does the unique id now change?

Reply:
It does not, the unique id was being passed to this function unnecessarily